suricata/update/main.py

```diff
@@ -1129,6 +1136,9 @@ def _main():
     
     update_parser.add_argument("--no-merge", action="store_true", default=False,
                                help="Do not merge the rules into a single file")
+    update_parser.add_argument("--offline", action="store_true", default=False,
```

Comment:
One more thing, I do not think you require a `default` while mentioning the `action` here, `store_true` means default is false and when specified, it shall be true. Read and try: https://docs.python.org/3/library/argparse.html#action